Barotrauma/BarotraumaShared/SharedSource/StatusEffects/StatusEffect.cs

```diff
@@ -1887,6 +1894,52 @@ protected void Apply(float deltaTime, Entity entity, IReadOnlyList<ISerializable
                 
                 TryTriggerAnimation(target, entity);
 
+                if (forceSayIdentifier != null) 
+                {
+                    Character targetCharacter = null;
+                    if (target is Character character)
+                    {
+                        targetCharacter = character;
+                    }
```

Comment:
The above can be simplified to 
`Character targetCharacter = target as Character;`